src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/Parser/PropertiesXmlParser.php

```diff
@@ -20,7 +20,7 @@
 use Symfony\Contracts\Translation\TranslatorInterface;
 
 /**
- * Parse properties structure from an XML file.
+ * @internal this class is not part of the public API and may be changed or removed without further notice
```

Comment:
this should make it in future easier if we need again changes in the parser or loader as we don't longer need to keep bc in mind.